quickemu

```diff
@@ -330,9 +330,16 @@ function vm_boot() {
     fi
     echo ", ${RAM_VM} RAM"
 
-    if [ "${guest_os}" == "macos" ] || [ "${guest_os}" == "windows" ] || [ "${guest_os}" == "windows-server" ]; then
+    if [ "${guest_os}" == "windows" ] || [ "${guest_os}" == "windows-server" ]; then
+        if [ "${RAM_VM//G/}" -lt 4 ]; then
+            echo "ERROR! The guest virtual machine has been allocated insufficient RAM to run Windows."
+            echo "       You can override the guest RAM allocation by adding 'ram=4G' to ${VM}"
```

Comment:
mybe "You can override the guest RAM allocation by adding 'ram=4G' to ${VM} 's config file"

Reply:
`${VM}` expands to the name of the config file.